app/models/transaction_type.rb

```diff
@@ -18,6 +18,7 @@ class TransactionType < ApplicationRecord
       maintenance_out
       legal_aid
     ],
+    null: %i[housing_benefit],
```

Comment:
I thought about how to approach this for quite a while, and I think this approach is reasonable —even though I'm not entirely satisfied.

I'm not convinced `null` is the right "operation", and I'm concerned it might add confusion. 

Having said that, I do think it conveys some meaning. Housing benefit transactions are neither credits or debits in the conventional sense, and we need to distinguish them somehow.

Reply:
Makes sense to me I think 👍

Reply:
It feels to me like it is fundamentally a credit and it will need to be sent to CFE as a `credit` operation as it currently stands.